test/chisel2.py

```diff
@@ -0,0 +1,216 @@
+"""
+A simple client that uses the Python ACME library to run a test issuance against
+a local Boulder server. Unlike chisel.py this version implements the most recent
+version of the ACME specification. Usage:
+
+$ virtualenv venv
+$ . venv/bin/activate
+$ pip install -r requirements.txt
+$ python chisel.py foo.com bar.com
+"""
+import json
+import logging
+import os
+import sys
+import signal
+import threading
+import time
+import urllib2
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography import x509
+from cryptography.hazmat.primitives import hashes
+
+import OpenSSL
+
+from acme import challenges
+from acme import client as acme_client
+from acme import crypto_util as acme_crypto_util
+from acme import errors as acme_errors
+from acme import jose
+from acme import messages
+from acme import standalone
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(int(os.getenv('LOGLEVEL', 0)))
+
+DIRECTORY = os.getenv('DIRECTORY', 'http://localhost:4001/directory')
```

Comment:
If we're storing this locally we might as well default to the same port/directory as Pebble uses.